src/org/opendatakit/briefcase/util/TransferFromODK.java

```diff
@@ -41,6 +40,8 @@
 
   private static final Logger log = LoggerFactory.getLogger(TransferFromODK.class);
 
+  private FilenameFilter xmlFilenameFilter = (dir, name) -> name.endsWith(".xml");
```

Comment:
We know it's a filename filter thanks to its `FilenameFilter` type and the member name feels redundant.

We can rename this member to something that tells us about the filtering criteria i.e "file ends with xml" or "has xml extension" or "it's an xml file"